src/renderer/index.js

```diff
@@ -18,6 +18,7 @@ import app from './app';
 import analytics from './analytics';
 
 const { autoUpdater } = remote.require('electron-updater');
+const appPageURL = 'lbry://?';
```

Comment:
Generally constants like this are in all caps `APP_PAGE_URL`

Reply:
fixed